mesonbuild/wrap/wrap.py

```diff
@@ -186,6 +187,13 @@ def get_directory(subdir_root: str, packagename: str) -> str:
         return wrap.directory
     return packagename
 
+# Wrapper to convert GitException to WrapException catched in interpreter.
```

Comment:
This should be a docstring. Also "caught" not "catched"

Reply:
ok, fixed.